Source/Diagnostics/BoostedFrameDiagnostic.cpp

```diff
@@ -844,20 +901,26 @@ writeMetaData ()
         HeaderFile << dt_snapshots_lab_ << "\n";    
         HeaderFile << gamma_boost_ << "\n";
         HeaderFile << beta_boost_ << "\n";
-        HeaderFile << dz_lab_ << "\n";
```

Comment:
Could you also make sure that none of these changes break the USE_HDF5 option?

Reply:
Would you have a test that I could use (and even add to automated tests) for that?